iib/web/models.py

```diff
@@ -241,6 +241,15 @@ def get_or_create(cls, name):
         return operator
 
 
+class BuildTag(db.Model):
```

Comment:
I'm sorry I missed this earlier. But could you please use an association table like`RequestArchitecture` to be consistent. Something like `RequestBuildTag`